### DISCUSSION TITLE
i3 4.24 released!

### DISCUSSION THREAD

**Discussion by stapelberg** [i3/i3#6294] [Announcements]
Release notes can be found at: https://i3wm.org/downloads/RELEASE-NOTES-4.24.txt

**Comment by rsommer** [comment]
Any ETA for Debian/Ubuntu packages? In the repository at https://debian.sur5r.net/i3 4.23 is still the latest version.

**Reply by stapelberg** [reply]
The ETA is whenever @sur5r gets around to updating the packages. If you’re in a hurry, you can check out the nightly autobuilder repository, see https://i3wm.org/docs/repositories
